server/src/lib.rs

```diff
@@ -279,26 +279,27 @@ impl InitData {
                                 trustify_module_graphql::endpoints::configure_graphiql(svc);
                             }),
                     );
-                    svc.app_data(web::Data::from(self.graph.clone()))
-                        .service(
-                            web::scope("/api")
-                                .wrap(new_auth(self.authenticator.clone()))
-                                .configure(|svc| {
-                                    trustify_module_importer::endpoints::configure(svc, db.clone());
-
-                                    trustify_module_fundamental::endpoints::configure(
-                                        svc,
-                                        db.clone(),
-                                        storage.clone(),
-                                    );
-                                }),
-                        )
-                        .configure(|svc| {
-                            // I think the UI must come last due to
-                            // its use of `resolve_not_found_to`
-                            #[cfg(feature = "ui")]
-                            trustify_module_ui::endpoints::configure(svc, &self.ui);
-                        });
+                    svc.app_data(web::Data::from(self.graph.clone())).service(
+                        web::scope("/api")
+                            .wrap(new_auth(self.authenticator.clone()))
+                            .configure(|svc| {
+                                trustify_module_importer::endpoints::configure(svc, db.clone());
+
+                                trustify_module_fundamental::endpoints::configure(
+                                    svc,
+                                    db.clone(),
+                                    storage.clone(),
+                                );
+                            }),
+                    );
+                    svc.app_data(web::Data::from(self.graph.clone())).service(
+                        web::scope("")
+                            .wrap(new_auth(self.authenticator.clone()))
```

Comment:
What problem are you addressing here? Why wrap the ui endpoints with auth that wasn't there before? What does that have to do with trailing slashes?

Reply:
@jcrossley3, the trailing slash is involved in  `trustify_module_ui::endpoints::configure()` and part of the confusion (at least for me) is that the UI is currently defined (in main) as part of https://github.com/trustification/trustify/blob/main/server/src/lib.rs#L282.

Reply:
What are you fixing? What specific URL breaks before this change and works after it?

Reply:
@jcrossley3, this is a reorganization fix I thought obvious. I can remove it.

Reply:
Forgive my stupidity. But I still want to know what you were fixing. It wasn't obvious to me.

Reply:
@jcrossley3, please don't be stupid ! ;) 
It's me who is not capable of explaining in plain words what I thought made sense and after all it might be out of scope. So I'll try in a separate PR....